### DISCUSSION TITLE
Suggestion: pinning "cards"

### DISCUSSION THREAD

**Discussion by BalinGrant** [JordanKnott/taskcafe#4] [Ideas]
I saw your reddit post about this project and looks really good. I have been using trello for a while now and I have always wanted to be able to run cards.  

For example, you have a list for ideas.  You want to add a card that you use as a description. The only thing is that this card sometimes gets moved around and being able to pin the card would fix this.

**Comment by JordanKnott** [comment]
An interesting idea, I'll add it to the roadmap!
